yang-module-versioning/draft-ietf-netmod-yang-module-versioning.txt

```diff
@@ -274,18 +250,19 @@ Internet-Draft    Updated YANG Module Revision Handling     October 2023
    non-linear development of YANG module and submodule revisions, so
    that they MAY have multiple revisions that directly derive from the
    same parent revision.  As per [RFC7950] and [RFC6020], YANG module
-
-
-
-Wilton, et al.           Expires April 19, 2024                 [Page 5]
-
-Internet-Draft    Updated YANG Module Revision Handling     October 2023
-
-
    and submodule revisions continue to be uniquely identified by their
    revision date, and hence all revisions of a given module or submodule
    MUST have unique revision dates.
 
+   However, using revsision dates alone to identify revisions of a YANG
+   module versioned with a branched revision history is likely to be
+   confusing because the relationship between module revisions is no
+   longer guaranteed to be chronologically ordered.  Instead, for
+   modules that may use a branched revision history, it is RECOMMENDED
+   to use a versioning scheme, such as [I-D.ietf-netmod-yang-semver],
+   where the revision identifier, e.g., the version label, better
+   describes the semantic relationship between the revisions.
+
    For a given YANG module revision, revision B is defined as being
```

Comment:
This whole bit about the derived relationship and consulting revision history was needed before because recommended-min went through the history. Do we still need this text?

Reply:
In light of the linearity, I don't think we do.  Maybe keep the bit about branching SHOULD use YANG Semver?

Reply:
I'm not sure what you mean by "In light of the linearity". We're still clarifying in Module Versioning that branching is allowed (e.g. the paragraph "This document clarifies [RFC7950] and [RFC6020] to explicitly allow non-linear...".

I guess these other paragraphs about derived relationship aren't strictly necessary here, but I somewhat feel they are still useful as conceptual info about branching.

Reply:
Discussed on Feb 27th meeting and we decided to keep this text (at least for now): even if it's not used for the extensions anymore the text does accurately describe the relationships.